packages/core/src/feed.ts

```diff
@@ -1,20 +1,53 @@
-import type { StreamMetadata, StreamState } from '@ceramicnetwork/common'
+import { CommitType, LogEntry, StreamMetadata, StreamState } from '@ceramicnetwork/common'
 import { Subject, type Observable } from 'rxjs'
 import { CommitID } from '@ceramicnetwork/streamid'
 import { StreamUtils } from '@ceramicnetwork/common'
 
+/**
+ * Describes all event types
+ */
+export enum EventType {
+  CREATE = 0,
+  ANCHOR = 1,
```

Comment:
toooooottaal nitpick that really doesn't matter at all, but I feel like I'll have an easier time remembering the values these represent if init/create is 0, data/update is 1, and time/anchor is 2.